modules/database/src/ioc/db/dbCa.c

```diff
@@ -369,9 +371,9 @@ void dbCaPause(void)
         epicsEventSignal(workListEvent);
     }
 }
-
-void dbCaAddLinkCallback(struct link *plink,
-    dbCaCallback connect, dbCaCallback monitor, void *userPvt)
+void dbCaAddLinkCallbackOpt(struct dbLocker *locker, struct link *plink,
```

Comment:
Re: previous comment, why are we taking the argument `locker` here as it is not used?